package.json

```diff
@@ -7,12 +7,20 @@
     ".": "./index.mjs",
     "./parse": {
       "worker": "./dist/cjs/parse.js",
-      "node": "./dist/cjs/parse.js",
+      "node": {
+        "require": "./dist/cjs/parse.js",
+        "import": "./dist/esm/parse.js",
+        "types": "./dist/esm/parse.d.ts"
```

Comment:
types should be defined below `browser`.

Reply:
Done.
`node` should also have `require` and `export`